### PR TITLE
MKLML build in scalable build, warnings, rnn and ssd operators

### PR DESCRIPTION
**Pull request by cjolivier01**

### CODE REVIEW

File: example/image-classification/train_mnist.py [hunk 1]
```diff
@@ -53,6 +53,8 @@ def get_mnist_iter(args, kv):
                         help='the number of classes')
     parser.add_argument('--num-examples', type=int, default=60000,
                         help='the number of training examples')
+    parser.add_argument('--num_epochs', type=int, default=20,
```
Comment (piiswrong) [inline example/image-classification/train_mnist.py]: add_fit_args will add this

File: src/operator/cast-inl.h [hunk 2]
```diff
@@ -0,0 +1,152 @@
+/*!
```
Comment (piiswrong) [inline src/operator/cast-inl.h]: this file was removed

File: example/image-classification/train_mnist.py (hunk 1 above)
Comment (cjolivier01) [inline example/image-classification/train_mnist.py]: Ok yeah I am having merge problems with a few pieces of code. I don;t know why.

File: src/operator/cast-inl.h (hunk 2 above)
Comment (cjolivier01) [inline src/operator/cast-inl.h]: same as above

**Comment by cjolivier01** [comment]
Ok, someone has moved all of these files to operator/contrib now...

**Pull request closed without merging**